src/catalogue.ts

```diff
@@ -117,6 +117,14 @@ export class CatalogueModule {
         });
     }
 
+    /**
+     * ItemDescription is either URI or MAQL expression
+     * https://github.com/gooddata/gdc-bear/blob/185.4/resources/specification/md/obj.res#L284
+     */
+    public getItemDescriptionsFromMdObject(projectId: string, mdObj: any, attributesMap: any) {
```

Comment:
please consider to make `bucketItemsToExecConfig` public and rename it to something more explicit :) for example `loadItemDescriptions` as it would be better to keep catalogue API consistent (see other calls starting with load...)

also please include arguments jn JSDoc

Reply:
Done :-)